### PR TITLE
Added ui test before loading extended widget plugin to avoid a crash

### PR DESCRIPTION
**Pull request by anaselli**

### CODE REVIEW

**Comment by anaselli** [comment]
This pull requests is from discussion in pull 74

**Comment by mvidner** [comment]
Thanks! This one is easy :)

Please add an entry to package/libyui.changes, just repeating the commit message is enough. (We should add a [CONTRIBUTING how-to like yast has](https://github.com/yast/yast-yast2/blob/master/CONTRIBUTING.md))

I'll use this for testing the https://ci.opensuse.org/view/libyui-staging/ setup.
The idea is that a human pushes a `staging` branch (manually, after a review, to guard against malicious PRs), Jenkins and OBS run tests, and then we are more confident to merge it.
(Right now the CI setup it is broken, the jobs have stopped watching the git repos, I'll fix that.)

**Comment by anaselli** [comment]
thanks @mvidner, i usually add it when the PR is accepted to avoid further conflicts :)

File: package/libyui.changes [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- Added ui test before loading extended widget plugin, to avoid
```
Comment (mvidner) [inline package/libyui.changes]: The dashy header with timestamp and email is missing. You can use `osc vc package/` to add it, or you can improvise one with 
```sh
dd if=/dev/zero status=none bs=1 count=67 | tr '\0' -; echo
echo "$(LANG=en_US date) - $USER@$(hostname -d)"
```

**Comment by mvidner** [comment]
Now the build setup is fixed and  https://ci.opensuse.org/view/libyui-staging/ and https://build.opensuse.org/project/monitor/devel:libraries:libyui:Staging should be useful.

**Comment by mvidner** [comment]
Thanks!

**Pull request merged**